src/pages/UserSignUp.tsx

```diff
@@ -90,11 +94,15 @@ export default function UserSignUpPage() {
                             register={register('ConfirmPassword')}
```

Comment:
please make sure to name form fields in `camelCase`

Reply:
Pascal case should only be use for class names, types, interfaces, enums and component names. Otherwise for variables and class members or props always use `camelCase`